client/app/telecom/pack/xdsl/pack-xdsl.html

```diff
@@ -27,7 +27,7 @@
                 </responsive-tab>
                 <responsive-tab data-state="telecom.pack.xdsl.modem"
                                 disabled="PackXdslCtrl.disabledModem"
-                                data-ng-if="PackXdslCtrl.content.status === 'active' || PackXdslCtrl.content.status === 'migration'"
+                                data-ng-if="['active', 'migration', 'upgradeOffer'].indexOf(PackXdslCtrl.content.status) > -1"
```

Comment:
Prefer doing this directly on the controller itself.

```js
return ['active', 'migration', 'upgradeOffer'].includes(PackXdslCtrl.content.status);
```

Reply:
Also `['active', 'migration', 'upgradeOffer']` could be a constant